app/views/shared/shop_show/_shop_about_card.html.haml

```diff
@@ -0,0 +1,17 @@
+#shop_about.card.card-block.col-md-4.col-md-offset-1
+  .text-xs-center
+  - if shop.profile_image?
+    = image_tag(shop.profile_image.url)
+  - else
+    %img.img-circle.image_center{:alt => "Profile picture",  src: "http://placehold.it/100x100"}
+    %p.about-title=shop.name
+  -# .text-xs-center
```

Comment:
I didn't mean the placeholder. The commented out section with social icons. And yes the placeholder too. You can set the default fallback image on that model attachment settings rather than checking here.